### PR TITLE
added extra output to url_parse

### PR DESCRIPTION
**Pull request by kelby-shelton**
added originalUrl as an output

### CODE REVIEW

File: custom_functions/url_parse.json [hunk 1]
```diff
@@ -16,34 +16,53 @@
...
+                "*"
+            ],
             "data_path": "fragment",
             "description": "The subcomponent of the resource which is identified after the hash sign."
+        },
+        {
+            "contains_type": [
+                "url"
+            ],
+            "data_path": "originalUrl",
```
Comment (philroyer-phantom) [inline custom_functions/url_parse.json]: Could you please use the variable name "input_url" so it is consistent with the lower case and underscore we use elsewhere, and also with the naming we use in other similar outputs like in the passthrough custom function?

**Review by philroyer-phantom — CHANGES_REQUESTED** [review]
minor style change only

File: custom_functions/url_parse.json (hunk 1 above)
Comment (kelby-shelton) [reply custom_functions/url_parse.json]: fixed in https://github.com/phantomcyber/playbooks/pull/17/commits/0bf33e6b8c69ee5827884235295276ec96ae12c7

File: custom_functions/url_parse.py [hunk 2]
```diff
@@ -6,12 +6,13 @@ def url_parse(url=None, **kwargs):
...
-        params: The parameters in the URL after the semicolon.
-        query: The query string of the URL after the question mark. Multiple parameters are not separated from each other.
-        fragment: The subcomponent of the resource which is identified after the hash sign.
+        scheme (CEF type: *): The scheme of the URL, such as HTTP, HTTPS, or FTP.
+        netloc (CEF type: domain): The network location of the URL, which is typically the hostname.
+        path (CEF type: *): The path to the resource after the first slash in the URL, such as "en_us/software/splunk-security-orchestration-and-automation.html".
+        params (CEF type: *): The parameters in the URL after the semicolon.
+        query (CEF type: *): The query string of the URL after the question mark. Multiple parameters are not separated from each other.
+        fragment (CEF type: *): The subcomponent of the resource which is identified after the hash sign.
+        originalUrl (CEF type: url): The original url that was parsed
```
Comment (philroyer-phantom) [inline custom_functions/url_parse.py]: I think this should be input_url to be consistent with above. Also, editing it outside of Phantom made it so its currently giving a warning saying "Function definition may be invalid" when I import it.

**Review by philroyer-phantom — CHANGES_REQUESTED** [review]

File: custom_functions/url_parse.py (hunk 2 above)
Comment (kelby-shelton) [reply custom_functions/url_parse.py]: I went back and made the code changes inside of Phantom. I also changed url to input_url and made .netloc to be cef data type "domain." Commit: 
https://github.com/phantomcyber/playbooks/pull/17/commits/b30af756d08b3e64a227b556b737f79891d52490

**Pull request merged**